From 311d1e5ad6278b6b08561c9163957f80c8e48af1 Mon Sep 17 00:00:00 2001
From: "Shaik, ShahinaX" <shahinax.shaik@intel.com>
Date: Mon, 5 Feb 2018 20:12:17 +0530
Subject: [PATCH 0663/1240] ASoC: Intel: Skylake: Update gain interface
 structure

This patch updates gain structure alignment as per
firmware interface requirement.

Change-Id: I67d509ec8aaff2f9f02d1ad830f03dca7fa50465
Signed-off-by: Shaik, ShahinaX <shahinax.shaik@intel.com>
Reviewed-by: Shaik, Kareem M <kareem.m.shaik@intel.com>
Reviewed-by: Kesapragada, Pardha Saradhi <pardha.saradhi.kesapragada@intel.com>
Reviewed-by: Kp, Jeeja <jeeja.kp@intel.com>
Reviewed-by: Kale, Sanyog R <sanyog.r.kale@intel.com>
Reviewed-by: Singh, Guneshwor O <guneshwor.o.singh@intel.com>
Reviewed-by: Prusty, Subhransu S <subhransu.s.prusty@intel.com>
Tested-by: Madiwalar, MadiwalappaX <madiwalappax.madiwalar@intel.com>
---
 sound/soc/intel/skylake/skl-tplg-interface.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/intel/skylake/skl-tplg-interface.h b/sound/soc/intel/skylake/skl-tplg-interface.h
index ef019a5..72544755 100644
--- a/sound/soc/intel/skylake/skl-tplg-interface.h
+++ b/sound/soc/intel/skylake/skl-tplg-interface.h
@@ -208,6 +208,7 @@ struct skl_gain_config {
 	u32 channel_id;
 	u32 target_volume;
 	u32 ramp_type;
+	u32 reserved;
 	u64 ramp_duration;
 } __packed;
 
-- 
2.7.4

